Packs/NetCalc/Integrations/IPCalc/IPCalc.py

```diff
@@ -0,0 +1,238 @@
+
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import traceback
+from typing import Dict, Any
+import ipcalc
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module() -> str:
+    ipcalc.Network('192.168.10.10/32')
+    return 'ok'
+
+
+def return_subnet_addresses_command(args: Dict[str, Any]) -> CommandResults:
+
+    subnet = args.get('subnet', None)
+
+    if not subnet:
+        raise ValueError('subnet not specified')
```

Comment:
If `subnet` arg is mandatory, no need to do this check.  (Suggestion for next integrations, no need to apply it now).

Reply:
sure